src/org/jitsi/impl/neomedia/rtp/StreamRTPManager.java

```diff
@@ -168,14 +171,17 @@ public void putResumableStreamRewriter(
     public ResumableStreamRewriter getResumableStreamRewriter(Long ssrc,
                                                               boolean create)
     {
-        ResumableStreamRewriter rewriter = ssrcToRewriter.get(ssrc);
-        if (rewriter == null && create)
+        synchronized (ssrcToRewriter)
```

Comment:
I'm pretty sure there's only a single thread accessing the map. The documentation is not clear about, sorry about that. Would you like me to add a note explaining this and why it is the case?